### PR TITLE
Chore improve helm scripts

### PR DESCRIPTION
**Pull request by LakshmiMekala**
Helm chart support for BE6.0

cc: @rameshpolishetti @vivekshende

### CODE REVIEW

File: cloud/kubernetes/helm/templates/db-configmapsa.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-{{- if and (eq .Values.persistentType "sharedAll" ) (eq .Values.omType "cache" ) }}
+{{- if and (eq .Values.bsType "store" ) (eq .Values.omType "cache" ) (eq .Values.storeType "RDBMS" ) }}
```
Comment (rameshpolishetti) [inline cloud/kubernetes/helm/templates/db-configmapsa.yaml:1]: @LakshmiMekala valid values for storeType would be - _**None / Oracle / SQL Server / DB2 / MySQL / PostgreSQL / ActiveSpaces 4x (AS4) / Apache Cassandra**_ but not _**RDBMS**_. 
Please amend the logic accordingly.

File: cloud/kubernetes/helm/values.yaml [hunk 2]
```diff
@@ -5,103 +5,84 @@
...
-omType: inmemory
-
-#Required parameter to be passed for sharedNothing, sharedAll
-persistentType: None
+cpType: minikube
+#-----------------------------------------
+cmType: AS2                     #cmType can be unclustered,AS2,FTL
+omType: cache                   #omType can be inmemory,store,cache
+bsType: sharedNothing         #BackingStore can be None sharedNothing, store
+storeType: None                 #storeType can be RDBMS, AS4, Cassandra
```
Comment (rameshpolishetti) [inline cloud/kubernetes/helm/values.yaml:13]: @LakshmiMekala valid values for storeType would be - None / Oracle / SQL Server / DB2 / MySQL / PostgreSQL / ActiveSpaces 4x (AS4) / Apache Cassandra

File: cloud/kubernetes/helm/templates/db-configmapsa.yaml (hunk 1 above)
Comment (LakshmiMekala) [reply cloud/kubernetes/helm/templates/db-configmapsa.yaml:1]: All the RDBMS related values ( Oracle / SQL Server / DB2 / MySQL / PostgreSQL) will be considered as RDBMS. For details please refer to helm Readme.md file

File: cloud/kubernetes/helm/values.yaml (hunk 2 above)
Comment (LakshmiMekala) [reply cloud/kubernetes/helm/values.yaml:13]: All the RDBMS related values ( Oracle / SQL Server / DB2 / MySQL / PostgreSQL) will be considered as RDBMS.

**Pull request merged**